ebean-ddl-generator/src/main/java/io/ebeaninternal/dbmigration/model/ModelDiff.java

```diff
@@ -125,6 +140,21 @@ public void compareTo(ModelContainer newModel) {
     // search for tables that are no longer used
     for (MTable existingTable : baseModel.getTables().values()) {
       if (!newTables.containsKey(existingTable.getName())) {
+        // Compounded foreign keys used in many-to-many mapping tables
```

Comment:
Without this code, the drop of the table would be done in wichever order the tables come along (alphabetically?), thus creating a foreign key conflict.

Reply:
So you mean `drop table_a; drop table_b` will fail, if table_b references table_a?
The other way round will work?

So we can either reorder the table or dropping the FKs first.
Reorder will not work in all cases (cycles - maybe provide as test?)
So dropping keys first should be the right way